### DISCUSSION TITLE
vue-easytable does not provide an export named 'default'

### DISCUSSION THREAD

**Discussion by BITRU** [nuxt/nuxt#17170] [Questions]
### Environment

------------------------------
- Operating System: `Darwin`
- Node Version:     `v18.6.0`
- Nuxt Version:     `3.0.0-rc.6`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `modules`, `runtimeConfig`, `unocss`, `experimental`, `css`, `build`, `colorMode`
- Runtime Modules:  `@pinia/nuxt@0.3.1`, `@vueuse/nuxt@8.9.4`, `@unocss/nuxt@0.44.7`, `@nuxtjs/supabase@0.1.19`, `@nuxtjs/color-mode@3.1.4`
- Build Modules:    `-`
------------------------------

### Reproduction

yarn add  vue-easytable

plugins/vue-easytable.client.ts - if not .client will fail

```
import VueEasytable from 'vue-easytable'
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VueEasytable)
})
```

```
defineNuxtConfig({  build: {
    transpile: ['vue-easytable'],
  },})
```

### Describe the bug

fail to run with error

Uncaught SyntaxError: The requested module '/_nuxt/node_modules/vue-easytable/libs/main.js?v=e7b2d731' does not provide an export named 'default' (at vue-easytable.client.ts:1:1)

**Comment by danielroe** [comment]
Looks like you might also need to add to `vite.optimizeDeps.include`: https://vitejs.dev/config/dep-optimization-options.html#optimizedeps-include.

**Reply by BITRU** [reply]
```
export default defineNuxtConfig({
...
  vite: { optimizeDeps: { include: ['vue-easytable',],},},
...
})
```

and remove:  transpile: ['vue-easytable'],

got it started. 

But now I have 
Failed to resolve component: VeTable
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 

keep trying to resolve it. If you have an idea what wrong will appreciate.
